src/tracks/MidiClip.cpp

```diff
@@ -185,13 +185,9 @@ TimePos MidiClip::beatClipLength() const
 
 
 
-Note * MidiClip::addNote( const Note & _new_note, const bool _quant_pos )
+Note * MidiClip::addNote(const Note& _new_note)
```

Comment:
Ideally remove the `_` prefix, too, though.
```suggestion
Note* MidiClip::addNote(const Note& _new_note)
```